static checks broken

```
level=warning msg="[runner/govet] govet error: assign: analysis skipped due to errors in package"
level=warning msg="[runner/govet] govet error: atomicalign: analysis skipped due to errors in package"
level=warning msg="[runner/govet] govet error: bools: analysis skipped due to errors in package"
level=warning msg="[runner/govet] govet error: buildtag: analysis skipped due to errors in package"
level=warning msg="[runner/govet] govet error: httpresponse: analysis skipped due to errors in package"
level=warning msg="[runner/govet] govet error: loopclosure: analysis skipped due to errors in package"
level=warning msg="[runner/govet] govet error: ctrlflow: analysis skipped due to errors in package"
level=warning msg="[runner/govet] govet error: lostcancel: failed prerequisites: ctrlflow@github.com/kata-containers/runtime/cli [github.com/kata-containers/runtime/cli.test]"
level=warning msg="[runner/govet] govet error: nilfunc: analysis skipped due to errors in package"
level=warning msg="[runner/govet] govet error: printf: analysis skipped due to errors in package"
level=warning msg="[runner/govet] govet error: shift: analysis skipped due to errors in package"
level=warning msg="[runner/govet] govet error: stdmethods: analysis skipped due to errors in package"
level=warning msg="[runner/govet] govet error: tests: analysis skipped due to errors in package"
level=warning msg="[runner/govet] govet error: unmarshal: analysis skipped due to errors in package"
level=warning msg="[runner/govet] govet error: unsafeptr: analysis skipped due to errors in package"
level=warning msg="[runner/govet] govet error: unusedresult: analysis skipped due to errors in package"
level=warning msg="[runner/govet] govet error: buildssa: analysis skipped due to errors in package"
level=warning msg="[runner/govet] govet error: nilness: failed prerequisites: buildssa@github.com/kata-containers/runtime/cli [github.com/kata-containers/runtime/cli.test]"
level=warning msg="[runner] Can't run linter govet: asmdecl: analysis skipped due to errors in package"
cli/delete.go:30:46: undeclared name: `name` (typecheck)
   If the container id is "ubuntu01" and ` + name + ` list currently shows the
                                             ^
cli/kata-env.go:474:9: undeclared name: `envCmd` (typecheck)
	Name:  envCmd,
	       ^
cli/list.go:88:16: undeclared name: `defaultRootDirectory` (typecheck)
(default: "` + defaultRootDirectory + `").
               ^
cli/list.go:109:38: undeclared name: `project` (typecheck)
			Usage: "display all available " + project + " information",
			                                  ^
cli/main.go:51:4: undeclared name: `projectPrefix` (typecheck)
`, projectPrefix, projectPrefix, projectURL)
   ^
cli/main.go:85:10: undeclared name: `configFilePathOption` (typecheck)
		Name:  configFilePathOption,
		       ^
cli/main.go:104:10: undeclared name: `showConfigPathsOption` (typecheck)
		Name:  showConfigPathsOption,
		       ^
cli/kata-check.go:293:9: undeclared name: `checkCmd` (typecheck)
	Name:  checkCmd,
	       ^
cli/main_test.go:1028:46: undeclared name: `version` (typecheck)
	err = grep(fmt.Sprintf(`%s\s*:\s*%s`, name, version), output)
	                                            ^
cli/main_test.go:903:40: undeclared name: `commit` (typecheck)
	testVersionString(assert, v, version, commit, unknown)
	                                      ^
cli/main_test.go:771:23: undeclared name: `defaultSysConfRuntimeConfiguration` (typecheck)
			assert.Equal(line, defaultSysConfRuntimeConfiguration)
```

--- a/.ci/static-checks.sh
+++ b/.ci/static-checks.sh
@@ -9,4 +9,8 @@ set -e
 cidir=$(dirname "$0")
 source "${cidir}/lib.sh"
 
+# Build kata-runtime before running static checks
+make -C "${cidir}/../"
+
+# Run static checks
 run_static_checks

